### PR TITLE
fix TVBS relax_attitude_control in tailsitter.cpp

### PR DESCRIPTION
**Pull request by kd0aij**
replaces #15195

Adds a new method relax_roll_and_yaw_controllers() to tailsitter.cpp and necessary accessors to AC_AttitudeControl.
Also moves tailsitter body-frame roll support into tailsitter.cpp.

### CODE REVIEW

**Comment by IamPete1** [comment]
I think this is also missing the relax in `hold_stabilize()`

File: libraries/AC_AttitudeControl/AC_AttitudeControl.h [hunk 1]
```diff
@@ -89,6 +89,29 @@ class AC_AttitudeControl {
     float get_accel_pitch_max() const { return _accel_pitch_max; }
     float get_accel_pitch_max_radss() const { return radians(_accel_pitch_max * 0.01f); }
 
+    // get and set target attitude quaternion
+    Quaternion get_attitude_target_quat() {return _attitude_target_quat; }
```
Comment (tridge) [inline libraries/AC_AttitudeControl/AC_AttitudeControl.h:93]: should return const ref

File: libraries/AC_AttitudeControl/AC_AttitudeControl.h [hunk 2]
```diff
@@ -89,6 +89,29 @@ class AC_AttitudeControl {
...
+    void set_attitude_target_quat(Quaternion &q) {_attitude_target_quat = q; }
+
+    // set target euler angles
+    void set_attitude_target_euler_angle(float x, float y, float z) {
```
Comment (tridge) [inline libraries/AC_AttitudeControl/AC_AttitudeControl.h:97]: should take Vector3f const ref

**Review by tridge — CHANGES_REQUESTED** [review]

**Comment by kd0aij** [comment]
@IamPete1 I cherry-picked the change to hold_stabilize

**Comment by kd0aij** [comment]
build/Pixracer/bin/arduplane.apj:    "image_size": 1429184
https://firmware.ardupilot.org/Plane/latest/Pixracer/arduplane.apg: "image_size": 1427616
delta: 1568

**Comment by kd0aij** [comment]
@Hwurzburg This is ready for flight testing. It looks OK in SITL with the RealFlight SkyCat2.

File: libraries/AC_AttitudeControl/AC_AttitudeControl.h [hunk 3]
```diff
@@ -89,6 +89,26 @@ class AC_AttitudeControl {
     float get_accel_pitch_max() const { return _accel_pitch_max; }
     float get_accel_pitch_max_radss() const { return radians(_accel_pitch_max * 0.01f); }
 
+    // get and set target attitude quaternion
+    const Quaternion& get_attitude_target_quat() {return _attitude_target_quat; }
```
Comment (rmackay9) [inline libraries/AC_AttitudeControl/AC_AttitudeControl.h:93]: these getters could be made const I suspect..

Comment (kd0aij) [reply libraries/AC_AttitudeControl/AC_AttitudeControl.h:93]: oops, that was a non-const duplicate... fixing it

**Comment by kd0aij** [comment]
@Hwurzburg @IamPete1 @tridge Hold off on this PR; Leonard and Randy strongly prefer https://github.com/ArduPilot/ardupilot/pull/15301

**Pull request closed without merging**